1-development/list-posts-by-rating/README.md

```diff
@@ -0,0 +1,78 @@
+# list-posts-by-rating Submission
+
+My submition for the requested InvoicExpress Challenge.
+
+## Tech Stack
+
+This solution uses Postgresql for its database.

```

Comment:
Why?

Reply:
I used postgreSQL because I had already used it in previous projects. I had no implementation or technological reasons behind this choice. It was just convenience.

Reply:
I didn't use any type of testing methodology like TDD or BDD for this submission, I was just programming  and making the requests to the server using POSTMAN and debugging it along the way.